jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -189,6 +189,7 @@ YUI.add('juju-topology-service', function(Y) {
           'stroke': '#19b6ee',
```

Comment:
Need to retain this as it apparently sets the stroke color when the charm is initially added to the canvas. Without this line, the charm will just have a normal grey stroke when first added.